### PR TITLE
459 ml document type endpoint

### PR DESCRIPTION
**Pull request by LightosTomJ**
DetermineDocumentType End-point created and debugged, with data persisting within SQL.

Refactoring & publishing of the database was required due to alterations made via [](https://github.com/data-summit/data-summit/pull/34/files). Test files did not exist in previous seed data.

### CODE REVIEW

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs [hunk 1]
```diff
@@ -29,13 +40,85 @@ public IActionResult IsRunning()
...
+                //{
+                //    var documentTypeClassification = await _classificationService.GetPrediction(blobUrl, "DocumentType", "Classification");
+                //    var documentTypeEnum = _dataSummitDocuments.DocumentType(documentTypeClassification.TagName);
+                //    var typeConfidence = Math.Round(documentTypeClassification.Probability, 3);
+
+                //    // Update blob metadata
+                //    List<KeyValuePair<string, string>> additionalMetaData = new List<KeyValuePair<string, string>>
+                //    {
+                //        new KeyValuePair<string, string>("DocumentType", documentTypeEnum.ToString()),
+                //        new KeyValuePair<string, string>("DocumentTypeConfidence", typeConfidence.ToString())
```
Comment (robbhudl) [inline DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:95]: Why all this commented out code?

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs [hunk 2]
```diff
@@ -29,13 +40,85 @@ public IActionResult IsRunning()
         [HttpPost("uploadFiles")]
         public async Task<IActionResult> UploadFiles(List<IFormFile> files)
         {
+            var uploadedFileURLs = new HashSet<string>();
             try
             {
-                var tasks = files.Select(file => _azureService.UploadDataToBlob(file));
```
Comment (robbhudl) [inline DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:34]: Why has this been removed?

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs (hunk 1 above)
Comment (LightosTomJ) [reply DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:95]: GitHub didn't include it for some reason. Do I perform another commit, revoke the pull request or other?

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs (hunk 2 above)
Comment (LightosTomJ) [reply DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:34]: Honestly, not sure upon review. I recall that the `UploadFiles` had a warning message due to the 'async' not being spotted by Visual Studio. I wanted to remove the error as I expect to copy this method on many occasions.

Comment (LightosTomJ) [reply DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:34]: The error is still there though! FFS.

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs (hunk 1 above)
Comment (robbhudl) [reply DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:95]: What do you mean by 
> GitHub didn't include it

What didn't GitHub include?

Either delete it or uncomment it if it is supposed to be in. That would be via a new commit

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs (hunk 2 above)
Comment (robbhudl) [reply DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:34]: What error?

Warnings are possibly fine - don't try to fix warnings in code you didn't write

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs (hunk 1 above)
Comment (robbhudl) [reply DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs]: Commented code has been deleted

File: DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs (hunk 2 above)
Comment (robbhudl) [reply DataSummitSolution/DataSummitWeb/Controllers/DocumentsController.cs:34]: There are no warnings, so code didn't need to change. Same pattern can be applied to new method

**Review by robbhudl — APPROVED** [review]
LGTM

**Pull request merged**